doc/manuals/admin/crate.md

```diff
@@ -0,0 +1,9 @@
+# Crate
+
+[**Crate**](https://crate.io) is Quantumleap's default backend where NGSI data will be persisted. Until Quantumleap implements its querying API, you can interact directly with Crate to query all the data QL has saved from the received notifications.
+
+If you followed the [Installation Guide](./index.md), you have already Crate running in a Docker container, ready to be used.
```

Comment:
If you followed the [Installation Guide](./index.md), you have a ready-to-use Crate instance running in a Docker container.

Reply:
it looks like you didn't implement my text edits.